cookie-store/cookieStore_set_arguments.https.any.js

```diff
@@ -285,3 +285,29 @@ promise_test(async testCase => {
   assert_equals(cookie.name, 'cookie-name');
   assert_equals(cookie.value, 'new-cookie-value');
 }, 'cookieStore.set with get result');
+
+promise_test(async testCase => {
+  // The maximum attribute value size is specified at https://wicg.github.io/cookie-store/#cookie-maximum-attribute-value-size.
+  const maximumAttributeValueSize = 1024
+  await cookieStore.delete('cookie-name');
+
+  await promise_rejects_js(testCase, TypeError, cookieStore.set(
+      { name: 'cookie-name',
+        value: 'cookie-value',
+        path: '/this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-it-this-string-has-1025-characters-in-/' }));
```

Comment:
Checking the length assumption here is a bit inconvenient.

Consider a more obvious formulation, such as `'/' + 'a'.repeat(1024)`

Reply:
Good point. Fixed!